Modules/Role/Http/Services/RoleService.php

```diff
@@ -80,23 +68,31 @@ public function show(): array
     public function details($requestData): array
     {
         $role = Role::findOrFail($requestData->id);
-        return [
-            'res' => true,
-            'msg' => '',
-            'data' => $role
-        ];
+
+        $permissions = $role->permissions;
```

Comment:
Instead of foreach loop, can't we just use the following query ?
$permissions = $role->permissions->pluck('id')->toArray();

Reply:
Fixed